rmi/servercode/ResImpl/ResourceManagerImpl.java

```diff
@@ -505,11 +505,24 @@ public boolean shutdown() throws RemoteException {
 
     public boolean vote(int transactionId) throws RemoteException, InvalidTransactionException, TransactionAbortedException {
         // Do something more here??
-        System.out.println("Sending NO vote for transaction: " + transactionId);
-        return false;
+        System.out.println("Sending YES vote for transaction: " + transactionId);

```

Comment:
Set this to "Always vote yes" for testing purposes. All of the logic stuff I wrote for committing (In this PR) is based on the premise that all RM's vote yes.

If they don't all vote yes, then we don't hit any of that logic anyways.